src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,8 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
 export const MovieList = () => (
-  <>Put the list here</>
+  <div className='movies'>
+    <MovieCard />
+  </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings

Reply:
Sorry, I didn't pay attention at this message. I have no idea, where I can use them at Component 'MovieList' now..

Reply:
It seems to me, I added these tags, but not sure, that in right place ..